### PR TITLE
Document smart proxy templates feature

### PR DESCRIPTION
**Pull request by stbenjam**
I never did re-open https://github.com/theforeman/theforeman.org/pull/313.  Documents the template proxy, tired of explaining it on IRC.

### CODE REVIEW

File: _includes/manuals/1.9/4.3.12_smartproxy_templates.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+In this chapter, we will describe how to setup a Smart Proxy to serve
+provisioning templates.
+
+The smart proxy is able to proxy template requests from hosts in isolated
+networks to the Foreman server, when the proxy also handles TFTP.
+
+<div class="alert alert-info">Generally, you want the templates to be available on HTTP as well as HTTPS.  When enabling HTTP on your smart proxy, ensure that other modules' configurations in /etc/foreamn-proxy/settings.d/*.yml are secure by setting :enabled: to https instead of true.</div>
```
Comment (domcleal) [inline _includes/manuals/1.9/4.3.12_smartproxy_templates.md]: Typo in the settings.d path, "foreamn"

File: _includes/manuals/1.10/4.3.12_smartproxy_templates.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+In this chapter, we will describe how to setup a Smart Proxy to serve
+provisioning templates.
+
+The smart proxy is able to proxy template requests from hosts in isolated
+networks to the Foreman server, when the proxy also handles TFTP.
+
+<div class="alert alert-info">Generally, you want the templates to be available on HTTP as well as HTTPS.  When enabling HTTP on your smart proxy, ensure that other modules' configurations in /etc/foreamn-proxy/settings.d/*.yml are secure by setting :enabled: to https instead of true.</div>
```
Comment (domcleal) [inline _includes/manuals/1.10/4.3.12_smartproxy_templates.md]: ditto

**Comment by stbenjam** [comment]
Fixed

**Comment by domcleal** [comment]
Thanks @stbenjam!

**Pull request merged**